pkg/apis/pipeline/v1/container_types.go

```diff
@@ -142,6 +142,15 @@ type Step struct {
 	// +optional
 	// +listType=atomic
 	Params Params `json:"params,omitempty"`
+	// Results declares StepResults produced by the step.
+	// This is field is at an alpha stability level and gated by "enable-step-actions" feature flag.
+	// It can be used in an inlined step when used to store results to
+	// $(step.results.resultName.path). When referencing StepActions,
+	// it cannot be used. The results declared by the StepActions will be stored here instead.
+	// Using it along side referencing StepActions will result in a validation error.
```

Comment:
Can you list the fields it's compatible with? Bonus points if you use https://tip.golang.org/doc/comment#doclinks!

Reply:
I think it is compatible with all the fields in a `step` except `step.Ref`. Not sure if I should list all the Step fields since `step.Ref` is the only field it is incompatible with.

Reply:
done.

Reply:
I added the docklinks for what it is not compatible with.